### DISCUSSION TITLE
MuJoCo event during ICRA 2023 in London, Thursday June 1st

### DISCUSSION THREAD

**Discussion by yuvaltassa** [google-deepmind/mujoco#875] [Announcements]
Hello MuJoCo community!

We are holding a social event at Google’s office in Kings Cross on Thursday evening, June 1st. It’s an opportunity to meet with fellow MuJoCo users and the development team in a relaxed and informal setting with views of the London skyline. We’ll be providing food and beverages and are looking forward to meeting you! Feel free to share this announcement with anyone who cares about physics simulation.

If you are interested in attending, please register your interest using [this form](https://rsvp.withgoogle.com/events/mujoco-icra-2023). Registration is required, and attendance will be confirmed in advance.

While we are not planning any formal talks or presentations, there will be some screens in the space. If you have a video or slide that you would like to showcase, please add a link to the registration form.

**Comment by FMCalisto** [comment]
This sounds like a fantastic event, @yuvaltassa! It's great to see opportunities for the [MuJoCo community](https://roboti.us/forum/index.php) to come together and share their experiences. The informal setting will surely foster engaging discussions and the exchange of innovative ideas. I'm particularly interested in seeing the various videos and slides that attendees will showcase. It's always inspiring to see the diverse ways in which physics simulation is being utilized.

**Comment by saikishor** [comment]
Hello!
Is the registration closed or something?. Because I haven't received any confirmation email.

**Reply by saran-t** [reply]
The event is unfortunately heavily oversubscribed: there were 300 signups but we only have capacity to host 100 people. If you didn't get a "See you tomorrow" email from us yesterday then unfortunately we weren't able to confirm your attendance. We're really sorry about this, it was difficult for us to anticipate ahead of time that the event would prove quite this popular!

A couple of us will be hanging around the tables ~outside N8/S8~ between S8 and N9 halls (next to Costa coffee) today and tomorrow, so please feel free to come and say hi if you want a chat!
